Signal/src/ViewControllers/ConversationView/Cells/OWSMessageFooterView.m

```diff
@@ -160,6 +185,19 @@ - (BOOL)isFailedOutgoingMessage:(ConversationViewItem *)viewItem
     return messageStatus == MessageReceiptStatusFailed;
 }
 
+- (BOOL)shouldShowDisappearingTimer:(ConversationViewItem *)viewItem
+{
+    OWSAssert(viewItem);
```

Comment:
TIOLI: we could move this logic into the viewItem: `viewItem.isDisappearingMessage`

Reply:
Done.